lab-justin/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "lab-justin",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "./node_modules/mocha/bin/mocha"
```

Comment:
Hi, what does "start:db": "mongod ~~~ do here?
i used to put server.js usually in "scripts"